src/CmsChangeLogBundle/src/esono/pkgcmschangelog/TCMSTableEditor/TCMSTableEditorChangeLog.class.php

```diff
@@ -140,56 +148,58 @@ protected function PostSaveHook(&$oFields, &$oPostTable)
     /**
      * computes differences between new and old values.
      *
-     * @param TIterator  $oFields    holds an iterator of all field classes from DB table with the posted values or default if no post data is present
+     * @param TIterator  $newFields  holds an iterator of all field classes from DB table with the posted values or default if no post data is present
      * @param TCMSRecord $oPostTable holds the record object of all posted data
      *
      * @return array
      */
-    protected function computeDifferences(&$oFields, &$oPostTable)
+    protected function computeDifferences($newFields, $oPostTable)
```

Comment:
Please do not change the method signature in a fix release.

Reply:
ok